### ISSUE TITLE
Update logo for accessability

### ISSUE THREAD

**Issue by BradyPlanden** [pybop-team/PyBOP#174]
Current logo renders black on black for dark reader usage. An updated logo for improved accessibility should be created. This [blog post](https://www.guvi.com/blog/designing-for-dark-mode-aesthetics/) could be used as a reference.
